packages/tools/examples/stackAnnotationTools/index.ts

```diff
@@ -185,6 +191,83 @@ addButtonToToolbar({
   },
 });
 
+const calibrationFunctions: Record<string, unknown> = {};
+
+const calibrations = [
+  {
+    value: 'Default',
+    selected: 'userCalibration',
+  },
+  {
+    value: 'User Calibration 0.5',
+    selected: 'userCalibration',
+    calibration: {
+      scale: 0.5,
+      type: Enums.CalibrationTypes.USER,
+    },
+  },
+  {
+    value: 'ERMF 2',
+    selected: 'userCalibration',
+    calibration: {
+      scale: 2,
+      type: Enums.CalibrationTypes.ERMF,
+    },
+  },
+  {
+    value: 'Projected 1',
+    selected: 'userCalibration',
+    calibration: {
+      // Bug right now in StackViewport that fails to reset
+      scale: 1,
+      type: Enums.CalibrationTypes.PROJECTION,
+    },
+  },
+  {
+    value: 'Error 1',
+    selected: 'userCalibration',
+    calibration: {
+      scale: 1,
+      type: Enums.CalibrationTypes.ERROR,
+    },
+  },
+  {
+    value: 'px units',
+    selected: 'applyMetadata',
+    metadata: {
+      '00280030': null,
+    },
+  },
+  {
+    value: 'Aspect 1:2',
+    selected: 'applyMetadata',
+    metadata: {
+      '00280030': { Value: [0.5, 1] },
+    },
+  },
+  {
+    value: 'Aspect 1:1',
+    selected: 'applyMetadata',
+    metadata: {
+      '00280030': { Value: [0.5, 0.5] },
+    },
+  },
+];
+const calibrationNames = calibrations.map((it) => it.value);
+
+addDropdownToToolbar({
+  options: { values: calibrationNames },
+  onSelectedValueChange: (newCalibrationValue) => {
+    const calibration = calibrations.find(
+      (it) => it.value === newCalibrationValue
+    );
+    if (!calibration) return;
+    const f = calibrationFunctions[calibration.selected];
+    if (!f) return;
+    f.apply(calibration);
+  },
+});
+
 /**
```

Comment:
I prefer a dedicated example to make it more searchable. Can you literally copy paste and remove pieces that you don't want?